paths/intro-to-github/00-welcome.md

```diff
@@ -1,66 +1,54 @@
 ---
-layout: intro-to-github
+layout: simple-class
 header:
   overlay_image: cover.jpeg
   overlay_filter: rgba(46, 129, 200, 0.6)
 title: Welcome
 permalink: /intro-to-github/
-facilitator: false
+next-page: /intro-to-github/01/
 sidebar:
   nav: "intro-nav"
----
-
-## Welcome
-
-If you are looking for a quick and fun introduction to GitHub, you've found it. This class is designed to get you started using GitHub in less than an hour.
-
-## What is GitHub?
-
-We are glad you asked! Many people stumble onto GitHub through one of the millions of Open Source projects it holds or because their employer or professor is using it. Why do these projects use GitHub?
+main-content: |
+  ## We're Glad You're Here :octocat:
 
-**At its heart, GitHub is a collaboration platform.**
+  This course is designed to get you started using GitHub in less than an hour.
 
-People around the world use GitHub to collaborate on projects ranging from software to policy documents and cookbooks. You can share your projects with the world and invite your friends to help, or you can keep your projects private and still have easy access wherever you are.
+  **In this course, you will learn how to**:
 
-**GitHub is also a powerful version control tool.**
+  - Use the GitHub.com website to complete the GitHub workflow
+  - Collaborate with ~~strangers~~ new friends on a fun project
+  - Add your pin to the map!
 
-While you are collaborating, GitHub uses Git (open source, distributed version control software) to keep track of every change made to your project.
+  <div id="map"></div>
 
-**GitHub is so much more.**
+  ## We Are Here to Help!
```

Comment:
> What are your thoughts on separating this into a second tab?

@hollenberry by a second tab do you mean a new page? We are already at 15 with the retro, do we want to add more? Could we find a more aesthetically pleasing way to display this information on this page? Maybe we could use something like [this image](http://betterevaluation.org/sites/default/files/StepsforTranslatingEvalFindingstoInfographics.jpg) that describes "steps for successfully completing the course" (e.g. do the activities, connect in gitter, review the tell me why sections, etc).